mne/tests/test_morph.py

```diff
@@ -0,0 +1,272 @@
+def test_stc_as_volume():
+    """Test previous volume source estimate morph."""
+    import nibabel as nib
+    inverse_operator_vol = read_inverse_operator(fname_inv_vol)
+
+    # Apply inverse operator
+    stc_vol = read_source_estimate(fname_vol, 'sample')
+
+    img = stc_vol.as_volume(inverse_operator_vol['src'], mri_resolution=True,
+                            dest='42')
+    with warnings.catch_warnings(record=True):  # nib<->numpy
+        t1_img = nib.load(fname_t1)
+    # always assure nifti and dimensionality
+    assert isinstance(img, nib.Nifti1Image)
+    assert img.header.get_zooms()[:3] == t1_img.header.get_zooms()[:3]
+
+    img = stc_vol.as_volume(inverse_operator_vol['src'], mri_resolution=False)
+
+    assert isinstance(img, nib.Nifti1Image)
+    assert img.shape[:3] == inverse_operator_vol['src'][0]['shape'][:3]
+
+    # Check if not morphed, but voxel size not boolean, raise ValueError
+    pytest.raises(ValueError, stc_vol.as_volume, inverse_operator_vol['src'],
+                  mri_resolution=(4., 4., 4.))
+
+
+@requires_nibabel()
+@requires_dipy()
+@testing.requires_testing_data
+def test_surface_vector_source_morph():
+    """Test surface and vector source estimate morph."""
+    inverse_operator_surf = read_inverse_operator(fname_inv_surf)
+
+    stc_surf = read_source_estimate(fname_smorph, subject='sample')
+    stc_surf.crop(0.09, 0.1)  # for faster computation
+
+    stc_vec = _real_vec_stc()
+
+    source_morph_surf = SourceMorph(subjects_dir=subjects_dir,
+                                    spacing=[np.arange(10242)] * 2,
+                                    src=inverse_operator_surf['src'])
+
+    assert isinstance(source_morph_surf, SourceMorph)
+
+    assert isinstance(SourceMorph(subjects_dir=subjects_dir)(stc_surf),
+                      SourceEstimate)
+
+    stc_surf_morphed = source_morph_surf(stc_surf)
+    stc_vec_morphed = source_morph_surf(stc_vec)
+
+    pytest.raises(ValueError, source_morph_surf.as_volume, stc_surf_morphed)
+
+    # check if correct class after morphing
+    assert isinstance(stc_surf_morphed, SourceEstimate)
+    assert isinstance(stc_vec_morphed, VectorSourceEstimate)
+
+    # check __repr__
+    assert isinstance(source_morph_surf.__repr__(), string_types)
+
+    # check loading and saving for surf
+    source_morph_surf.save(os.path.join(tempdir, '42.h5'))
+
+    source_morph_surf_r = read_source_morph(os.path.join(tempdir, '42.h5'))
+
+    assert (all([read == saved for read, saved in
+                 zip(sorted(source_morph_surf_r.__dict__),
+                     sorted(source_morph_surf.__dict__))]))
+
+    # check wrong subject correction
+    stc_surf.subject = None
+    assert isinstance(source_morph_surf(stc_surf), SourceEstimate)
+
+
+@requires_nibabel()
+@requires_dipy()
+@pytest.mark.slowtest
+@sample.requires_sample_data
+def test_volume_source_morph():
+    """Test volume source estimate morph, special cases and exceptions."""
+    import nibabel as nib
```

Comment:
why nib? nibabel is pretty enough.

Reply:
or `from nibalbel import Nifti1Image, Nifti2Image`

Reply:
we use nib everywhere.

Its also the standard import name of this lib: http://nipy.org/nibabel/gettingstarted.html